lib/pymedphys/tests/dicom/test_coords.py

```diff
@@ -58,7 +61,7 @@ def run_xyz_function_tests(coord_system):
     }
 
     for orient, dicom in test_ds_dict.items():
-        test_xyz = coords.xyz_axes_from_dataset(dicom, coord_system)
+        test_xyz = xyz_axes_from_dataset(dicom, coord_system)
```

Comment:
I personally significantly prefer the previous approach. It makes it clear that this function was not defined within this file, but instead it exists on an imported module.

Reply:
Fair enough!